requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Optionals
 aiofiles
-databases[sqlite]
+asyncpg
```

Comment:
Is `asyncpg` intentional here?

Reply:
Oh, moved it there since it was in setup.py, but obsolete since e396f07.

One good reason for havings reqs in one place I guess.